RELEASE/scripts/sl_ascend.ash

```diff
@@ -943,6 +943,7 @@ void maximize_hedge()
 	element third = ns_hedge3();
 	if((first == $element[none]) || (second == $element[none]) || (third == $element[none]))
 	{
+		doHottub();
```

Comment:
Should we maybe check if we have flared nostrils before calling `doHottub` so we don't waste a hottub use on nothing if we don't have it?

Reply:
My reasoning here was "there might be other bad effects that are worth removing here", but looking at the wiki that doesn't appear to be true. Will guard with a `have_effect()` check.